diff --git a/src/Fl_win32.cxx b/src/Fl_win32.cxx
index 0895ca417..834d82677 100644
--- a/src/Fl_win32.cxx
+++ b/src/Fl_win32.cxx
@@ -551,18 +551,18 @@ class Fl_Win32_At_Exit {
 public:
   Fl_Win32_At_Exit() {}
   ~Fl_Win32_At_Exit() {
-    fl_free_fonts(); // do some Windows cleanup
-    fl_cleanup_pens();
-    OleUninitialize();
-    if (fl_graphics_driver) fl_brush_action(1);
-    fl_cleanup_dc_list();
+    //fl_free_fonts(); // do some Windows cleanup
+    //fl_cleanup_pens();
+    //OleUninitialize();
+    //if (fl_graphics_driver) fl_brush_action(1);
+    //fl_cleanup_dc_list();
     // This is actually too late in the cleanup process to remove the
     // clipboard notifications, but we have no earlier hook so we try
     // to work around it anyway.
-    if (clipboard_wnd != NULL)
-      fl_clipboard_notify_untarget(clipboard_wnd);
+    //if (clipboard_wnd != NULL)
+    //  fl_clipboard_notify_untarget(clipboard_wnd);
 #if USE_GDIPLUS
-    Fl_GDIplus_Graphics_Driver::shutdown();
+    //Fl_GDIplus_Graphics_Driver::shutdown();
 #endif
   }
 };
